### PR TITLE
Make overlay wrapper full screen and transparent when shading is disabled (T758340)

### PR DESCRIPTION
**Pull request by mrjohn0011**

### CODE REVIEW

File: js/ui/overlay/ui.overlay.js [hunk 1]
```diff
@@ -1286,7 +1281,12 @@ var Overlay = Widget.inherit({
     _getContainer: function() {
         var position = this._position,
             container = this.option("container"),
-            positionOf = position ? position.of || window : null;
+            positionOf = null;
+
+        if(!container && position) {
+            var isEvent = !!(position.of && position.of.type === "positionEvent");
```
Comment (DokaRus) [inline js/ui/overlay/ui.overlay.js]: > position.of.type === "positionEvent"

There's no such thing. You can adopt logic from [animation/position](https://github.com/DevExpress/DevExtreme/blob/19_2/js/animation/position.js#L328)

File: testing/tests/DevExpress.ui.widgets/overlay.tests.js [hunk 2]
```diff
@@ -808,6 +808,19 @@ testModule("position", moduleConfig, () => {
         assert.strictEqual($overlayWrapper.css("position"), devices.real().ios ? "absolute" : "fixed");
     });
 
+    test("wrapper should have 100% width and height when shading is disabled", (assert) => {
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets/overlay.tests.js:811]: Add the `shading` option change test at runtime. And check the `pointer-events` value

**Review by DokaRus — CHANGES_REQUESTED** [review]
Should we update or remove `_renderShadingDimensions` from [Popover](https://github.com/mrjohn0011/DevExtreme/blob/overlay_shading/js/ui/popover.js#L539)?

File: styles/widgets/common/list.less [hunk 3]
```diff
@@ -51,7 +51,6 @@
     cursor: pointer;
     display: table;
     width: 100%;
-    table-layout: fixed;
```
Comment (DokaRus) [inline styles/widgets/common/list.less]: it's gonna cause a problem with the ListItem badge

**Review by DokaRus — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**